### PR TITLE
adding resources #486

### PR DESCRIPTION
**Pull request by andy5995**
I've listed each resource in the two commit messages, along with tags I've added.

I wasn't sure where to put the Mental Health humor link; for now, it's listed under _Communites_

### CODE REVIEW

File: doc/pages/services.json [hunk 1]
```diff
@@ -98,7 +98,19 @@
   {
     "name":"US Department of Veteran's Affairs | Mental Health",
     "link":"https://www.mentalhealth.va.gov/gethelp.asp",
-    "tags":["free", "suicide prevention", "homeless", "directory", "veterans"],
+    "tags":["free", "suicide prevention", "homeless", "directory", "veterans", "government"],
     "languages":["en"]
+  },
+  {
+    "name":"Resource Links - Disabled Veterans National Foundation",
```
Comment (andy5995) [inline doc/pages/services.json]: Oops I forgot to add (US) to the end here. Should I correct that? And also the link above (but it starts with "US").

Comment (julianguyen) [reply doc/pages/services.json]: Maybe rephrase to "Disabled Veterans National Foundation Resources (US)"?

File: doc/pages/communities.json [hunk 2]
```diff
@@ -82,5 +82,17 @@
     "link":"http://emotionsanonymous.org/",
     "tags":["free", "support_groups"],
     "languages":["en"]
+  },
+  {
+    "name":"In the Wake of Suicide...trying to understand",
```
Comment (julianguyen) [inline doc/pages/communities.json]: I would put these two resources under Education, since Communities is more for resources that are support groups or meetups :) These are great!

**Comment by andy5995** [comment]
Thanks @julianguyen 😺
